<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepressionAnalysis</class>
 <widget class="QDialog" name="DepressionAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="enable_btn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Disabled</string>
   </property>
  </widget>
  <widget class="QPushButton" name="history_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cloud Backup</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>700</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="HeaderImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Header.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>365</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Developed By: Muhammad Ehsan, Khizar Amin, Ahsan Munir, Rizwan Ahmed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>470</y>
     <width>110</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>FYP FAST-NUCES 2018</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Security Status:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="history_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>661</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="watchdog">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WatchDog</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
